main/http_server/axe-os/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Should this be checked in. What is cSpell?

Reply:
Yes, this is so VSCode settings are the same for this project. cSpell is spelling and grammar extension, it'll pick up on words like toastr which are valid words.